### PR TITLE
Simplify the design by using the main branch for all wordles (resolves #25)

### LINKED ISSUE #25: Only use the main branch for all wordles
## Is your feature request related to a problem?
With current design, when creating a new wordle, a new GitHub branch is created to read/write wordle data. This branch is also deployed to Netlify as a standalone website. The downside of this design includes:

1. Creating a new wordle needs to wait for the netlify deploy to complete. The wait time is around 30 seconds.
2. Using multiple branches increases the maintenance work by multiple times. For example, when merging a pull request, it needs to be merged into more branches.
3. All branch deploys have same netlify endpoint and front-end UI but dealing with different data, which is unnecessary.

## Describe the solution you'd like
Only use the main branch to handle all wordles.

### PR DESCRIPTION
**Pull request by cindyli**
This pull request is dependent on https://github.com/inclusive-design/git-ops-api/pull/29 which should be reviewed and merged first. When the new git-ops-api is published to npm, [package.json](https://github.com/cindyli/inverted-wordles/blob/simplify-arch/package.json#L22) should be updated to point to the new release.

All production wordles have been migrated into this pull request. After merging this pull request, these data should be double checked to make sure to include all new changes after the migration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +103,7 @@ environment variables can be added directly to it as follows:
 
 ```env
 REPOSITORY_URL=YOUR-REPOSITORY-URL
+REPOSITORY_BRANCH=YOUR-REPOSITORY-BRANCH
```
[the remark this answers is not in the document]
Comment (cindyli) [reply README.md]: You are right, @amb26. Netlify defined "BRANCH" is available on both the production site and local instances launnched by using `netlify dev`. I've removed the requirement of defining an extra environment variable. Switched to use Netlify defined "BRANCH". The change is pushed up at https://github.com/inclusive-design/inverted-wordles/pull/26/commits/40df99e3f3cfc30643d6b09faad30b755a5490c9.

File: src/js/login_handler.js [hunk 2]
```diff
@@ -11,25 +11,24 @@ document.addEventListener("DOMContentLoaded", () => {
...
-        for (let i = 0; i < inputElements.length; i++) {
-            if (options.inputFieldNames.includes(inputElements[i].getAttribute("name"))) {
-                if (enabled) {
-                    inputElements[i].removeAttribute("disabled");
-                } else {
-                    inputElements[i].setAttribute("disabled", "disabled");
-                }
+    // Enable/Disable input text fields
+    const inputElements = wordlesArea.querySelectorAll("input");
+    for (let i = 0; i < inputElements.length; i++) {
```
Comment (amb26) [inline src/js/login_handler.js:18]: inputElements.forEach is possibly a touch clearer here

File: src/js/login_handler.js [hunk 3]
```diff
@@ -11,25 +11,24 @@ document.addEventListener("DOMContentLoaded", () => {
...
+            } else {
+                inputElements[i].setAttribute("disabled", "disabled");
             }
         }
+    }
 
-        parentContainer.querySelector(options.selectors.deleteButton).disabled = enabled ? false : true;
+    // Enable/Disable delete buttons
+    const delButtons = wordlesArea.querySelectorAll(options.selectors.deleteButton);
+    for (let i = 0; i < delButtons.length; i++) {
```
Comment (amb26) [inline src/js/login_handler.js:30]: delButtons.forEach(delButton => delButton.disabled = !isLoggedIn)

File: src/js/manage_wordles_utils.js [hunk 4]
```diff
@@ -149,120 +121,3 @@ inverted_wordles.manage.findWordleRowByBranchName = function (wordlesAreaSelecto
...
-    // append the new row to the wordle list
-    document.querySelector(wordlesAreaSelector).insertAdjacentHTML("beforeend", newWordleRow);
-};
-
-/**
- * Append the wordle row that is in the process of deploy to the wordle list.
- * @param {String} wordlesAreaSelector - The wordles area selector.
- * @param {WordleValues} wordleValues - Values to be rendered for this Wordle.
- */
-inverted_wordles.manage.appendInDeployWordleRow = function (wordlesAreaSelector, wordleValues) {
```
Comment (amb26) [inline src/js/manage_wordles_utils.js:191]: Well this huge heap of red is a great loss :)

**Comment by amb26** [comment]
I tried testing this pull out using the previous workflow, and although I had set REPOSITORY_URL I got an interesting failure on startup:

"Error at fetching wordles: Error at getDirInfo(): No commit found for the ref simplify-arch"

Looks like the README might need updating in order to account for new testing protocol - Cheers!

File: src/js/create_wordle.js [hunk 5]
```diff
@@ -11,53 +11,78 @@ inverted_wordles.manage.bindCreateEvent = function (options) {
     createButton.addEventListener("click", evt => aria.openDialog(options.createDialogId, evt.target.id, options.createCancelId));
```
[the remark this answers is not in the document]
Comment (cindyli) [reply src/js/create_wordle.js:11]: The modal dialog popped up when creating a new question has been removed in https://github.com/inclusive-design/inverted-wordles/pull/27. Ready for a look.

**Pull request closed without merging**